common/src/TrenchBroomApp.cpp

```diff
@@ -112,6 +112,37 @@ namespace TrenchBroom {
                 return;
             }
 
+            // initialize stylesheets
+            QString styleSheetString;
+
+            // load base stylesheet
+            QFile baseStyle = QFile(IO::pathAsQString(IO::SystemPaths::appDirectory() + IO::Path("styles/base.qss")));
```

Comment:
const

Reply:
The `QTextStream` constructor requires a non-const `QFile`, so making this one const won't work.

Reply:
I see.